.config

```diff
@@ -0,0 +1,12 @@
+#! /bin/bash
+
+# Mandatory settings
+SETTING_EXTERNAL_HOST='localhost.localdomain'
+SETTING_ZULIP_ADMINISTRATOR='admin@example.com'
+
+# Optional secrets config
+POSTGRES_PASSWORD='REPLACE_WITH_SECURE_POSTGRES_PASSWORD'
+MEMCACHED_PASSWORD='REPLACE_WITH_SECURE_MEMCACHED_PASSWORD'
+RABBITMQ_DEFAULT_PASS='REPLACE_WITH_SECURE_RABBITMQ_PASSWORD'
+REDIS_PASSWORD='REPLACE_WITH_SECURE_REDIS_PASSWORD'
+SECRETS_secret_key='REPLACE_WITH_SECURE_SECRET_KEY'
```

Comment:
What do you think about changing this to a `.ini` format, like `/etc/zulip/zulip.conf`?  We can use `crudini` to make programmatic edits to that type of file.  

I guess the main downside with something like that is that we'll need to add more dependencies.

Reply:
Seems like the cleaner option whether we go with bash or python.